commandSql_datasource=Az adatforr\u00E1s
commandSql_description=Egy egyszer\u0171 sql fel\u00FCleten a mell\u00E9kelt adatforr\u00E1s.
databaseDatasourceTypeName=adatb\u00E1zis
exceptionCouldNotExecuteStmt=Lek\u00E9rdez\u00E9si nem hajthat\u00F3 v\u00E9gre\: {0}
exceptionCouldNotOpenDatasource=Nem siker\u00FClt megnyitni adatforr\u00E1s\: {0}
exceptionCouldNotPrepareStmt=Lek\u00E9rdez\u00E9st nem lehetett k\u00E9sz\u00EDteni\: {0}
